compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -458,12 +412,45 @@ static bool areOpsAggresiveFusable(
       })) {
     return false;
   }
+  return true;
+}
 
-  // Finally only fuse if the `ins` operand can be properly bufferized.
-  // TODO(#10498): Handle the multi-result case.
```

Comment:
What's the status of https://github.com/iree-org/iree/issues/10498?

I think we're able to handle the case for a while. Can you verify it and close the issue?